application/atomic-appcontracts/erc20/src/main/solidity/GpactERC20Bridge.sol

```diff
@@ -18,15 +18,15 @@ import "../../../../../../common/openzeppelin/src/main/solidity/token/ERC20/IERC
 import "../../../../../../functioncall/interface/src/main/solidity/CrosschainFunctionCallInterface.sol";
 import "../../../../../../common/openzeppelin/src/main/solidity/access/AccessControl.sol";
 import "../../../../../../common/openzeppelin/src/main/solidity/security/Pausable.sol";
-import "../../../../../../functioncall/interface/src/main/solidity/HiddenParameters.sol";
 import "./presets/LockableERC20PresetMinterPauser.sol";
```

Comment:
`gradle solCheckFormat` fails for this PR, can we run `gradle solFormat` before merging please.
Local `gradle build` should catch this, but this should also have been caught by CircleCI, I will address that in a separate PR.